03.macos/05.solving-problems/01.install-cert/docs.en.md

```diff
@@ -18,6 +18,6 @@ To manually install the certificate into Firefox-like browser:
   
   4. Navigate to "Authorities" tab and click on "Import..." button
   
-  5. Select file `/Library/Application Support/com.adguard.Adguard/NfApiConfiguration/SSL/Adguard Personal CA.cer`  or just download it from http://local.adguard.org/cert using a Chromium-based browser (e.g Google Chrome or new Edge) and with a HTTPS-filtering running in AdGuard
+  5. Select file `/Library/Application Support/com.adguard.Adguard/AdguardCore/Adguard Personal CA.cer`  or just download it from http://local.adguard.org/cert using a Chromium-based browser (e.g Google Chrome or new Edge) and with a HTTPS-filtering running in AdGuard
```

Comment:
File path is `/Library/Application Support/com.adguard.mac.adguard/AdguardCore/Adguard Personal CA.cer` in fact, or `/Library/Application Support/com.adguard.mac.adguard-setapp/AdguardCore/Adguard Personal CA.cer` for _setapp_ application version